test/integration/main_test.go

```diff
@@ -262,6 +295,14 @@ deletion_timeline:
 			if diff := cmp.Diff(wantPayload, gotPayload, protocmp.Transform()); diff != "" {
 				t.Errorf("gotPayload unexpected diff (-want,+got):\n%s", diff)
 			}
+			cmpOpts := []cmp.Option{
+				protocmp.Transform(),
+				protocmp.IgnoreFields(&v1alpha1.GitHubSource{}),
```

Comment:
If you ignore this, what fields do you compare the diff in the `cmp.Diff(wantGithubSource, gotPmapEvent.GetGithubSource(), cmpOpts...)`

Reply:
Typo, I meant ignoreUnexported. Already updated the code.